source/precalculus/source/08-TE/05.ptx

```diff
@@ -12,6 +12,464 @@
+          </p>
+        </statement>
+        <answer>
+          <p>
+            Students should draw a right triangle so that the base of the triangle is <m>67</m> meters, as it represents the shadow of the Space Needle.
+          </p>
+        </answer>
+      </task>
+        
+        <task>
+        <statement>
+          <p>
+            If the angle of elevation from the tip of the shadow to the top of the Space Needle is <m>70</m><degree/>, which trig function could you use to find the height of the Space Needle?
+          </p>
+        </statement>
+        <answer>
+          <p>
+            The tangent function would be the best to use because the opposite of the given angle (angle of elevation) is the height of the Space Needle and the shadow of the Space Needle would be the adjacent side.
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            How tall is the Space Needle (to the nearest meter)?
+          </p>
+        </statement>
+        <answer>
+          <p>
+            Approximately <m>184</m> meters.
+          </p>
+        </answer>
+      </task>
+    </activity>
+
+    <activity xml:id="right-triangles-practice">
+      <introduction>
+        <p>
+          Use <xref ref="def-angle-of-depression-and-elevation"/> and your knowledge of right triangles to solve each of the following. It might be helpful to draw a diagram to represent the situation before solving.
+        </p>
+      </introduction>
+      <task>
+        <statement>
+          <p>
+            Sarah's kite is flying above a field at the end of <m>65</m> meters of string. If the angle of elevation to the kite measures <m>70</m>, how high is the kite above Sarah's head?  
+          </p>
+        </statement>
+        <answer>
+          <p>
+            <m>61</m> meters
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            Standing on a cliff <m>380</m> meters above the sea, Sean sees an approaching ship and measures its angle of depression, obtaining <m>9</m> degrees. How far from shore is the ship (to the nearest meter)? 
+          </p>
+        </statement>
+        <answer>
+          <p>
+            <m>2,399</m> meters
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            A <m>14</m>-foot ladder is used to scale a <m>13</m>-foot wall. At what angle of elevation (to the nearest degree) must the ladder be situated in order to reach the top of the wall? 
+          </p>
+        </statement>
+        <answer>
+          <p>
+            <m>68</m><degree/>
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            One of the legs of a right triangle is twice as long as the other, and the perimeter of the triangle is <m>28</m>. Find the lengths of all three sides (to the nearest whole number). 
```

Comment:
This one doesn't require students to find any angles, so it seems out of place in this section.